src/lib/network/clients/save_network.rb

```diff
@@ -40,7 +39,10 @@ def adjust_for_network_disks(file)
         Installation.destdir
       )
 
-      return unless is_disk_in_network
+      unless is_disk_in_network
```

Comment:
[According to SUSE's Ruby style guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md#statements) we should not use `unless` in multiline format. I.e. this is considered acceptable
```ruby
do_whatever unless is_disk_in_network
```
But this is not :-(
```ruby
unless is_disk_in_network
  do_whatever
end
```
So I guess we should do this to be compliant:
```ruby
if !is_disk_in_network
  log.info("...")
  return
end
```